### PR TITLE
Maven test - Fix locale issue

### PR DESCRIPTION
**Pull request by stonio**
Ensure Maven tests are OK for any language/file encoding.

### CODE REVIEW

File: src/test/java/net/bramp/ffmpeg/ReadmeTest.java [hunk 1]
```diff
@@ -68,12 +69,12 @@ public void testGetMediaInformation() throws IOException {
 
     FFmpegFormat format = probeResult.getFormat();
     String line1 =
-        String.format("File: '%s' ; Format: '%s' ; Duration: %.3fs", format.filename,
+        String.format(Locale.US, "File: '%s' ; Format: '%s' ; Duration: %.3fs", format.filename,
```
Comment (bramp) [inline src/test/java/net/bramp/ffmpeg/ReadmeTest.java]: Can we just call Locale.setDefault before the test, to avoid cluttering up the example

Comment (stonio) [inline src/test/java/net/bramp/ffmpeg/ReadmeTest.java]: Method `Locale#setDefault(Locale)` changes locale for the instance of the JVM. I would rather use a specific locale only for this test ; maybe via a local field ?

**Comment by stonio** [comment]
We should also ensure tests are OK whatever the OS language or file encoding. I have not found the way to configure these in Travis-CI.

**Comment by bramp** [comment]
I agree tests should work regardless of the Locale. However, in this case we are depending on the specific formatting, and I want to keep the documentation simple.

I would only want Locale.setDefault for this one test. Can we now Locale.setDefault in a setup, and undo it in a teardown method?

**Comment by stonio** [comment]
I have added some cumbersome code to use `Locale#setDefault(Locale)`.

However, this code is not thread-safe and can have side-effects with [Parallel Test Execution](http://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html).

**Comment by bramp** [comment]
You make a good case that this code is no longer thread-safe. So perhaps we annotate with @NotThreadSafe, or perhaps we just change the test to use format(locale,...), and keep the README the simple.

**Pull request merged**